Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+React is a jvascript framework that renders a virtual DOM. DOM manipulation is problematic, and can cause bugs when things happen on the app. React takes objects and causes them to interact with eachother smoothly when things change on the app. 
 
 1.  What does it mean to _think_ in react?
+To think in React is to think about how the app works in realtion to the UI, and how the app changes as the user progresses through the app. 
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
-
+The only difference I could find on google between class/stateful component is that every class comonent is stateful, but not every stateful component is a class component.
+A functional component is just a javascript function. 
+Presentational components are concerned with how things look. 
 1.  Describe state.
-
+State is an object that is limited to the components, and can be updated when it needs to change.It's scope is limited to that component.
 1.  Describe props.
+A parent component can pass data to its children components as props, but the child can not modify that data.Unlike state, props scope can be passed down.

```

Comment:
both state and props can be passed down. Scope involves functions more than objects.